src/test/java/com/epam/tc/api/hw3/TestData.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.tc.api.hw3;
+
+import org.testng.annotations.DataProvider;
+
+public class TestData {
+
+    @DataProvider
```

Comment:
Are you sure you need these data providers? It seems that they use data from one equivalence class, so these checks are redundant